conferences/abstracts/OSF19_Demo_submission.md

```diff
@@ -0,0 +1,91 @@
+# OSFair2019 Demo Submission 
+Open Science Fair 2019, September 16-18, Porto, Portugal
+
+*For Authors: remove text in italics and replace with your own content.*
+
+## The Turing Way: A handbook for reproducible data science 
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
+
+## Abstract
+
+*Summary of your proposal; maximum 300 words. The abstract should be a concise summary of the demo content and the relation to the overall objective of the conference. Please copy and paste this into the submission system at the time of submission.*
+
+Reproducible research is necessary to ensure that scientific work can be trusted. Funders and publishers are beginning to require that publications include access to the underlying data and the analysis code. The goal is to ensure that all results can be independently verified and built upon in future work, which is sometimes easier said than done. Sharing these research outputs means understanding data management, library sciences, software development, and continuous integration techniques: skills that are not widely taught or expected of academic researchers and data scientists. 
+
+*The Turing Way* is a handbook to support students, their supervisors, funders and journal editors in ensuring that reproducible data science is "too easy not to do". It includes training material on version control, analysis testing, and open and transparent communication with future users, and builds on Alan Turing Institute case studies and workshops. This project is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way. 
+
+During this session, we will lead a collaborative review of the content so far and show Open Science Fair participants how they can contribute their knowledge to make it even better going forwards. This demo relates to the overall theme of the conference, as the *Turing Way* provides the tools to improve research habits in a self-contained handbook. It will also ensure that PhD students, postdocs, PIs and funding teams know which parts of the "responsibility of reproducibility" they can affect, and what they should do to nudge data science to being more efficient, effective and understandable.
```

Comment:
I think this sounds great - thank you for submitting the application!!

I don't know if the first two paragraphs really emphasise the community angle as much as they could for this audience. I feel like the sentence about it being openly developed needs to be brought front and centre for this event. What do you think?